eagle-xl/scripts/bh_masses.py

```diff
@@ -0,0 +1,116 @@
+"""
+Makes a BH dyn. mass vs. subgrid mass plot. Uses the swiftsimio library.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftsimio import load
+
+from unyt import unyt_quantity
+from matplotlib.colors import LogNorm
+from matplotlib.animation import FuncAnimation
+
+mass_bounds = [6e3, 6e9]
+
+
+def get_data(filename):
+    """
+    Grabs the data (masses in Msun).
+    """
+
+    data = load(filename)
+
+    mass_sub = data.black_holes.subgrid_masses.to("Msun")
+    mass_dyn = data.black_holes.dynamical_masses.to("Msun")
+    mass_seed = unyt_quantity(
+        float(data.metadata.parameters["EAGLEAGN:subgrid_seed_mass_Msun"]), "Msun"
```

Comment:
```suggestion
        float(data.metadata.parameters.get("EAGLEAGN:subgrid_seed_mass_Msun", 0.0)), "Msun"
```
will not break in the case where this script is accidentally ran on a different sub-grid model.